java/client/src/org/openqa/selenium/chrome/ChromeDriverCommand.java

```diff
@@ -24,4 +24,7 @@
   private ChromeDriverCommand() {}
 
   static final String LAUNCH_APP = "launchApp";
+  static final String SEND_COMMANDS_FOR_DOWNLOAD_CHROME_HEAD_LESS
+      = "sendCommandForDownloadChromeHeadLess";
```

Comment:
Nit: `Headless` is one word, not two, and so doesn't need camelcasing in this way.